docs/guides/Quick Start/02-product-management.md

```diff
@@ -0,0 +1,547 @@
+    "productNumber": {
+      "description": "Any random number given to product",
+      "type": "string"
+    },
+    "stock": {
+      "description": "Availability of stock",
+      "type": "string"
+    },
+    "taxId": {
+      "description": "ID of [tax](../../../adminapi.json/components/schemas/Tax)",
+      "type": "string"
+    },
+    "price": []
+      "currencyId": {
+      "description": "ID of [currency](../../../adminapi.json/components/schemas/Currency)",
+      "type": "string"
+      },
+      "gross": {
+      "description": "Gross price of a product",
+      "type": "string"
+      },
+      "net": {
+      "description": "Net price of a product",
+      "type": "string"
+      },
+      "linked": {
+      "description": "If set to true, net price is automatically calculated based on gross price and stored tax rate",
+      "type": "boolean"
+      }
+  }
+}
+```
+
+The `Productfeatureset` is a default template that extends from `productEntity` to insert all information for a new product.
+
+Here `linked` parameter takes boolean values. This signifies that if `priceForCurrency.linked` is true, the net price is calculated from the gross price based on tax type by the `calculatePrice()` method.
+
+The following payload examples contain UUIDs for various entities such as currencies, tax rates, manufacturers, or properties. These IDs are different on each system and must be adjusted accordingly.
+
+## Category
+
+All products are categorized in the catalogue. 
+
+Below is a sample request to create a category:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/category",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  },
+  "body": {
+    "displayNestedProducts": true,
+    "type": "page",
+    "productAssignmentType": "product",
+    "name": "Home"
+    }
+  }
+```
+
+```json json_schema
+{
+  "type": "object",
+  "description": "Parameters for category creation",
+  "properties": {
+    "displayNestedProducts": {
+      "description": "Specify `true` to display nested products else `false` ",
+      "type": "boolean"
+    },
+    "type": {
+      "description": "Types of category - `folder`, `link`, `page` ",
+      "type": "string",
+      "Example": " `product`, `category` "
+    },
+    "productAssignmentType": {
+      "description": "Accepts values `product`, `product-stream` for products added explicitly and added via Dynamic product group respectively",
+      "type": "string"
+    },
+    "name": {
+      "description": "Name of the category",
+      "type": "string"
+    }
+  }
+}
+```
+
+## Product assignment
+
+Every product must be assigned to a category for its display. It can be explicitly assigned or can be assigned as a Dynamic Product Group to a category.
+
+Let us assign the test product to the *Home* category created earlier:
+
+```text
+PATCH https://localhost/api/category/{category-id}
+```
+
```

Comment:
```suggestion
```

Redundant, as it's already shown with the next codeblock

Before change:
![image](https://user-images.githubusercontent.com/7780750/203090292-4ecea0d4-00f2-47c7-aad9-84e06d0f20b5.png)

After change:
![image](https://user-images.githubusercontent.com/7780750/203090401-46dce833-a607-4dc0-8a7d-474118071499.png)